plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -594,7 +594,9 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             pcth.runBeforeExecution(forExecutionHooks);
             args = (List<String>)forExecutionHooks.get("args");
             Set<String> types = new HashSet<>((List<String>) forExecutionHooks.get("types"));
-            userProperties.put("types", String.join(",", types));
+            PluginCompatTesterConfig afterHookExecution = ((PluginCompatTesterConfig) forExecutionHooks.get("config"));
+            mconfig.userProperties.putAll(afterHookExecution.retrieveMavenProperties());
+            mconfig.userProperties.put("types", String.join(",", types));
```

Comment:
```suggestion
            PluginCompatTesterConfig afterHookExecution = ((PluginCompatTesterConfig) forExecutionHooks.get("config"));
            userProperties.putAll(afterHookExecution.retrieveMavenProperties());
            userProperties.put("types", String.join(",", types));
```

to minimize diff as per https://github.com/jenkinsci/plugin-compat-tester/blob/fb7470582e30709fc67d190e5c48ede12ba8beed/plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java#L576